02-feature-engineering.Rmd

```diff
@@ -1037,19 +1037,19 @@ Of course, the more splines you are tuning, the more computationally expensive i
 
 ## Interactions
 
-An interaction occurs when the slope, or relation, between one variable and the outcome *depends upon* a second variable. This is also often referred to as *moderation* or a *moderating variable*. For example, consider data that look like this
+An interaction occurs when the slope, or relation, between one variable and the outcome *depends upon* a second variable. This is also often referred to as *moderation* or a *moderating variable*. For example, consider data that look like this:
 
 ```{r }
 benchmarks <- read_csv(here::here("data", "benchmarks.csv"))
 
 ggplot(benchmarks, aes(rdg_fall, rdg_spr)) +
   geom_point(color = "gray70") +
   geom_smooth(aes(color = ell),
```

Comment:
Just a heads up that this color palette isn't super colorblind-friendly (it's hard for me to distinguish between the colors for "Active" and "Monitor")

<img width="732" alt="Screen Shot 2020-11-09 at 8 36 21 PM" src="https://user-images.githubusercontent.com/32279312/98629117-01df8d00-22cd-11eb-8e7f-0aab88f7a53f.png">

Reply:
Good to know! I hadn't checked for that at all. If you go to the `_common.R` file you can see the code that sets those global color scales. If you want to switch it to a different one that works better that would be great! Ideally one that can handle a decent number of categories. You might want to look at the paletteer pkg for ideas.

Reply:
Sweet, I'll look into that!